agent/src/main/scala/za/co/absa/atum/agent/dispatcher/HttpDispatcher.scala

```diff
@@ -32,6 +32,7 @@ class HttpDispatcher(config: Config) extends Dispatcher with Logging {
   private val currentApiVersion = "/api/v1"
   private val createPartitioningEndpoint = Uri.unsafeParse(s"$serverUrl$currentApiVersion/createPartitioning")
   private val createCheckpointEndpoint = Uri.unsafeParse(s"$serverUrl$currentApiVersion/createCheckpoint")
+  private val createMetadataEndpoint = Uri.unsafeParse(s"$serverUrl$currentApiVersion/saveMetadata")
```

Comment:
I propose to keep the naming conventions consistent, so instead of `Metadata` let's use `AdditionalData` here and also on all other places. It's gonna be easier to understand the intentions / references / dictionary. What do you think?

Reply:
still TODO